code/_onclick/hud/hud.dm

```diff
@@ -19,10 +19,10 @@
 /datum/hud
 	var/mob/mymob
 
-	var/hud_shown = 1			//Used for the HUD toggle (F12)
-	var/inventory_shown = 1		//the inventory
-	var/show_intent_icons = 0
-	var/hotkey_ui_hidden = 0	//This is to hide the buttons that can be used via hotkeys. (hotkeybuttons list of buttons)
+	var/hud_shown 			= 1			//Used for the HUD toggle (F12)
+	var/inventory_shown 	= TRUE		//the inventory
```

Comment:
Weird additional spaces?

Reply:
That's VSCode being dumb again.. For some reasons tabs come out weird only with VSCode, but with anything else it doesn't do it. I'll take a look.

Reply:
You shouldn't be using tabs for spacing, it's not just VSC that displays them inconsistently.

Reply:
done!

Reply:
Also, can't help it, I'm used to not having to care between spaces and tabs since I just usually have them all replaced with spaces in my ide. DM is just messing with me XD